### PR TITLE
feat: add clusterRole

### PR DESCRIPTION
**Pull request by kranurag7**
Signed-off-by: GitHub <noreply@github.com>

**Purpose of PR?**:
- add ClusterRole 

**Does this PR introduce a breaking change?**
- I think it shouldn't bring any breaking changes. I have tested it on a 3 node GKE cluster. 

**If the changes in this PR are manually verified, list down the scenarios covered:**:
- logs of kubearmor components 
- karmor probe 
- policy enforcement 

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_
#900 

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: deployments/get/objects.go [hunk 1]
```diff
@@ -39,12 +64,11 @@ func GetClusterRoleBinding(namespace string) *rbacv1.ClusterRoleBinding {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      clusterRoleBindingName,
-			Namespace: namespace,
```
Comment (daemon1024) [inline deployments/get/objects.go:42]: Was namespace redundant here?

Comment (kranurag7) [reply deployments/get/objects.go:42]: It's a `clusterrolebinding` which is applied across the cluster, so thought of removing this namespace field from this one. Tested this on GKE, nothing is breaking, so far. 
Having said that, leaving the namespace field as it as is also fine.

**Review by nyrahul — APPROVED** [review]
Thanks for handling this. LGTM.

**Comment by nyrahul** [comment]
Hey @kranurag7 , can you please rebase to latest main and push?

**Pull request merged**